pyproject.toml

```diff
@@ -155,22 +154,35 @@ manylinux-x86_64-image = "manylinux2014"
 manylinux-aarch64-image = "manylinux2014"
 musllinux-x86_64-image = "musllinux_1_1"
 # RUNNER_OS is a GitHub Actions specific env var; define it here so it works on Cirrus CI too
-environment = {RUNNER_OS="Linux"}
+[tool.cibuildwheel.linux.environment]
+RUNNER_OS="Linux"
+PKG_CONFIG_PATH="/project/.openblas"
+LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/project/.openblas/lib"
 
 [tool.cibuildwheel.macos]
 # universal2 wheels are not supported (see gh-21233), use `delocate-fuse` if you need them
 archs = "x86_64 arm64"
 test-skip = "*_universal2:arm64"
-environment = {RUNNER_OS="macOS"}
+# Not clear why the DYLD_LIBRARY_PATH is not passed through from the environment
+repair-wheel-command = [
+  "export DYLD_LIBRARY_PATH=$PWD/.openblas/lib",
+  "echo DYLD_LIBRARY_PATH $DYLD_LIBRARY_PATH",
+  "delocate-wheel --require-archs {delocate_archs} -w {dest_dir} -v {wheel}",
```

Comment:
I wonder if we should specifically exclude the openblas lib from the delocate call for Sonoma wheels?

Reply:
It would be cleaner, yes. Setting `DYLD_LIBRARY_PATH` should be harmless, since 
- `$PWD/.openblas/lib` should not exist
- the build should use Accelerate, so delocate should not find any need for OpenBLAS.

Reply:
I was thinking along the lines of it being a reinforcement for knowing the wheel is using Accelerate.